engine/installation/linux/repo_files/oracle/docker-ol6.repo

```diff
@@ -1,27 +1,27 @@
 [docker-main]
 name=Docker Repository
-baseurl=https://yum.dockerproject.org/repo/main/oraclelinux/$releasever/
```

Comment:
If you renamed your directory (or created a symlink) on your YUM server to `7Server` and `6Server`, you could keep the `$releasever` variable and simplify back to a single `docker.repo` file for both Oracle Linux 6 and 7.